src/components/table.ts

```diff
@@ -51,14 +51,16 @@ export module Component {
       this.classed("table", true);
       rows.forEach((row, rowIndex) => {
         row.forEach((component, colIndex) => {
-          this.addComponent(rowIndex, colIndex, component);
+          if (component != null) {
```

Comment:
Maybe we should test the behavior of `this.addComponent(row, col, null)` if the cell is already occupied.